calyx-backend/src/verilog.rs

```diff
@@ -151,7 +373,61 @@ impl Backend for VerilogBackend {
                 "File not found: {}",
                 file.as_path_string()
             ))
-        })
+        })?;
+
+        if check_library_needed(ctx) {
+            let handlers: Vec<Box<dyn LibraryHandlerTrait>> =
+                vec![Box::new(HardFloatHandler)];
+            // Special libraries (like HardFloat) are needed, run Morty to pickle the files
+            let library_bundle =
+                build_library_bundle(ctx, &temp_file, &handlers)?;
+
+            let file_list = derive_file_list(ctx, &temp_file, &handlers)?;
+            let syntax_trees = morty::build_syntax_tree(
+                &file_list, false, // By default, don't strip comments
+                false, // By default, don't ignore unparseable
+                true, // By default, propagate defines from first files to the following files
+                false, // By default, don't force sequential
+            )
+            .unwrap();
+            let top_module = ctx.entrypoint.to_string();
+            let _pickled = morty::do_pickle(
```

Comment:
why are you not using the result here?

Reply:
because the pickled file will be in-place written to `temp_file`; and the result could be used to write the manifest: https://docs.rs/morty/latest/morty/fn.write_manifest.html, which I don't think we need it here